app/plugins/Wallet/js/helpers.js

```diff
@@ -5,7 +5,7 @@ const IPC = require('ipc');
 // Library for arbitrary precision in numbers
 const BigNumber = require('bignumber.js');
 // Ensure precision
-BigNumber.config({ DECIMAL_PLACES: 24 });
+BigNumber.config({ DECIMAL_PLACES: 30 });
```

Comment:
My thought was the largest unit is a MegaSiacoin via the send transaction so someone would be able to send 1.000000000000000000000000000001 MS to send 1 MS + 1 hasting without rounding